deployments/k8s/khaas/conf/ingestor/kubehound.yaml

```diff
@@ -0,0 +1,80 @@
+#
+# Default Datadog KubeHound configuration
+# NOTE: this is optimized for large clusters in the Datadog environment
+#
+
+# K8s collector configuration
+collector:
+  # Type of collector to use
+  type: live-k8s-api-collector
+
+  # Live collector configuration
+  live:
+    # Rate limit of requests/second to the Kubernetes API
+    rate_limit_per_second: 150
+
+    # Number of pages to buffer
+    page_buffer_size: 50
```

Comment:
just realized that the name is a bit confusing. I thought it would represent the number of elements per page, but the comment tells me otherwise.
Not an issue for this pr though :)

Reply:
yes k8s official naming here 😅